config/schedule.yml

```diff
@@ -0,0 +1,14 @@
+generate_random_events:
+  cron: "57 * * * *"
+  class: "EventGenerationJob"
+  queue: default
+
+sending_events_list:
+  cron: "6 18 * * * Asia/Kolkata"
+  class: "EventsListJob"
+  queue: default
+
+sending_event_reminder:
+  cron: "0 * * * *"
+  class: "EventReminderJob"
+  queue: default
```

Comment:
Where are we using this???

Reply:
This is used to schedule cron jobs for sidekiq, where we have to give time for running with job name and queue type, and when we start our sidekiq it would run these jobs in background